sonarqube/assets/configuration/spec.yaml

```diff
@@ -89,6 +93,57 @@ files:
             type: array
             items:
               type: string
+      - name: projects
```

Comment:
I think this config option could be more visible if we also write a small snippet about auto-discovering all projects in the Sonarqube `README`, WDYT?

Reply:
Updated README.md